index.js

```diff
@@ -0,0 +1,2 @@
+const nombreUsuario = prompt ("ingrese su nombre")
+alert (`Hola ${nombreUsuario}!`);
```

Comment:
aca esta perfecto aplicado!
```suggestion
alert (`Hola ${nombreUsuario}!`);
```